source/iface/unsampled-image-accessor.rst

```diff
@@ -24,125 +70,205 @@ Unsampled image accessors
 
 ::
 
-   template <typename dataT, int dimensions, sycl::access::mode accessmode,
-             sycl::access::target accessTarget = sycl::access::target::global_buffer,
-             sycl::access::placeholder isPlaceholder = sycl::access::placeholder::false_t>
-   class accessor;
-
-Description
+  class host_unsampled_image_accessor;
```

Comment:
This is missing the template parameters, which makes it hard to understand the text above.  Please add the full declaration including template parameters here.

Reply:
Does it match your idea with commit : https://github.com/KhronosGroup/SYCL_Reference/pull/103/commits/3b7703b9b546c60fc54f01f0d13297f08a2769bc

Reply:
Fixed with commit : https://github.com/KhronosGroup/SYCL_Reference/pull/103/commits/c0ddbd63cf04bf73fb8e85357dcf5aecf0e4805a